### PR TITLE
Concordance

### PR DESCRIPTION
**Pull request by edmondac**

### CODE REVIEW

**Review by mike-allaway — APPROVED** [review]
lgtm (albeit with my limited knowledge of R!)

**Review by edmondac — COMMENTED** [review]
Looks good to me too

File: ChangeLog [hunk 1]
```diff
@@ -1,3 +1,13 @@
+2021-04-29  Mike Allaway <m.j.allaway@bham.ac.uk> and Andrew Edmondson <a.c.edmondson@bham.ac.uk>
+
+    Version 1.2-0
```
Comment (edmondac) [inline ChangeLog]:
```suggestion
    Version 2.0
```

File: DESCRIPTION [hunk 2]
```diff
@@ -2,30 +2,39 @@ Encoding: UTF-8
 Package: CorporaCoCo
 Type: Package
 Title: Corpora Co-Occurrence Comparison
-Version: 1.1-0
-Date: 2017-11-22
+Version: 1.2-0
```
Comment (edmondac) [inline DESCRIPTION]:
```suggestion
Version: 2.0
```

File: NEWS [hunk 3]
```diff
@@ -1,7 +1,11 @@
-CHANGES IN VERSION 1.1-0:
+CHANGES IN VERSION 1.2-0:
```
Comment (edmondac) [inline NEWS]:
```suggestion
CHANGES IN VERSION 2.0:
```

**Comment by violawiegand** [comment]
This looks good as far as I can tell.

The documentation will still need some updating, however, for example:

1. The "proof_of_concept" and "faq" vignettes still refer to the old version of the package and use some deprecated functions such as `surface_coco()`. 
2. In relation to [this issue](https://github.com/birmingham-ccr/CorporaCoCo/issues/2), the "intro" vignette (and the corp_concordance help entry) should clarify the role of collocates. In the `corp_concordance()` help file, an example of a concordance would also be helpful (or it could refer the user to the "intro" vignette), and the documentation should explicitly explain the difference between running a concordance on a `corp_text` object vs. a `corp_cooccurrence` object (and the implications thereof).
3. The documentation for `corp_rbindlist()` is rather short and could therefore include more explanation and perhaps a brief example.
4. The documentation for `corp_text()` mentions a "details:" section which seems to be missing, at least from this page, so might require cross-referencing to another page ("For details of constructing your own tokens see details.").

**Pull request merged**